docs/framework/litellm.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Using Composio With LiteLLM"
+sidebarTitle: "LiteLLM"
+icon: "people-group"
+description: "Integrate Composio with LiteLLM and seamlessly interact with external apps"
+---
+
+## Star A Repository on Github
+In this example, we will use LiteLLM to star a repository on Github using Composio Tools
+
+<Steps>
+<Step title="Install Packages">
+```bash Python
+ pip install composio_openai litellm
+```
+</Step>
+<Step title="Import Libraries, Initialize ComposioToolSet & Create Router">
+```bash Python
+from composio_openai import ComposioToolSet, App
+from litellm.router import Router
+
+toolset = ComposioToolSet()
+
+router = Router(
+    model_list=[
+        {
+            "model_name": "anthropic/claude-3-5",
+            "litellm_params": {
+                "model": "claude-3-opus-20240229",
```

Comment:
The model name "claude-3-opus-20240229" appears to be outdated or incorrect. Please verify the correct model identifier from Anthropic's documentation and update accordingly. Also, consider adding a note about model version compatibility.